### PR TITLE
Add DFGM commands to the ground station and tests for them

### PR DESCRIPTION
**Pull request by DanielSacro**

### CODE REVIEW

File: src/groundStation/system.py [hunk 1]
```diff
@@ -1183,7 +1183,23 @@ def __init__(self):
...
+                                '###############################\r\n'+
+                                'Core_Voltage': '>f4',
+                                'Sensor_Temperature': '>f4',
+                                'Reference_Temperature': '>f4',
+                                'Board_Temperature': '>f4',
+                                'Positive_Rail_Voltage': '>f4',
+                                'Input_Voltage': '>f4',
+                                'Reference_Voltage': '>f4',
+                                'Input_Current': '>f4',
+                                'Reserved_1': '>f4',
```
Comment (tganley) [inline src/groundStation/system.py:1199]: What are these? Are they on the satellite side as well? If so could you add them to the housekeeping google sheet?

Comment (tganley) [reply src/groundStation/system.py:1199]: Also I though these were integers

File: src/groundStation/system.py [hunk 2]
```diff
@@ -3007,5 +3023,58 @@ def __init__(self):
...
+                        }
+                    },
+                    'DFGM_GET_HK': {
+                        'subPort': 3,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b',
+                                'Core Voltage': '>f',
+                                'Sensor Temperature': '>f',
```
Comment (tganley) [inline src/groundStation/system.py:3064]: Are these not stored as integers on the satellite?

File: src/groundStation/system.py (hunk 1 above)
Comment (DanielSacro) [reply src/groundStation/system.py:1199]: That's the DFGM HK data. They should all already be on the google sheet right now aside from the "Reserved" values, which I'll probably add to the sheets tomorrow. Also, you're right! They're supposed to be uint16_t, so I'll change them all from >f4 to >u2 sometime tomorrow

File: src/groundStation/system.py (hunk 2 above)
Comment (DanielSacro) [reply src/groundStation/system.py:3064]: Yes you are correct. They should be >u2 for uint16_t instead of >f, which I'll fix sometime tomorrow!

**Pull request merged**